tests/iso-8859-2-hungarian/auto-apro.hu.xml

```diff
@@ -1,474 +0,0 @@
-<?xml version="1.0" encoding="iso-8859-2" ?>
```

Comment:
The tag removing & filtering (with nested tags too) with using regex (and without it) I've implemented in some my projects (without BeatifulSoap or xmlElementTree), but here we need also some simple heuristic, because some XML and HTML files contains something like:
(&lt;acronym title="HyperText Markup Language"&gt;HTML&lt;/acronym&gt; és &lt;acronym title="Cascading Style Sheet"&gt;CSS&lt;/acronym&gt;)